kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -100,6 +100,8 @@
 <script>
 
   const actions = require('../../actions');
+  const coreActions = require('core-actions');
```

Comment:
These two requires are problematic, because this will bundle this code separately into the management plugin bundle, in addition to it being bundled into the core app.